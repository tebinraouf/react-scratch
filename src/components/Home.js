import React from "react";
import Toronto from "../static/images/toronto.jpg";
import "../static/styles/home.css";

console.log(process.env.NODE_ENV);

class Home extends React.Component {
	componentDidMount () {
		fetch(`${API_URL}toronto`)
			.then(result => result.json())
			.then(json => console.log(json));
	}
	render () {
		return (
			<div>
				<h1>Hello Universe!</h1>
				<img src={Toronto} width="200"/>
			</div>
		);
	}
}
export default Home;