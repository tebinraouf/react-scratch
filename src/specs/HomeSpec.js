import React from "react";
import Home from "../components/Home";

import renderer from "react-test-renderer";

describe("<Home />", () => {
	test("Should render correctly", () => {
		const component = renderer.create(<Home />);
		const tree = component.toJSON();

		expect(tree).toMatchSnapshot();
	});
});