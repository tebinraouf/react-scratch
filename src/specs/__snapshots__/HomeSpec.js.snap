// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> Should render correctly 1`] = `
<div>
  <h1>
    Hello Universe!
  </h1>
  <img
    src="test file stub"
    width="200"
  />
</div>
`;
